samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/ClientApp/src/components/tab.css

```diff
@@ -0,0 +1,20 @@
+.text-label {
```

Comment:
Perhaps, I am missing something, but I don't see where this class name is used.
Same for .ed-label and .ed-container below